sopel_waifu/waifu.json

```
@@ -1511,11 +1560,28 @@
         "Yowane Haku"
     ],
     "WataMote": [
-        "Tomoko Kuroki"
+        "Kuroki Tomoko",
+        "Naruse Yuu",
+        "Satozaki Kiko",
+        "Nemoto Hina",
+        "Megumi Imae",
```

This is in Given Family order, by accident.


```suggestion
        "Imae Megumi",
```